InAppMessagingDisplay/Example/FiamDisplaySwiftExample/BridgeHeader.h

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
This file should be part of the public headers so that static libraries work out of the box for anyone installing the pod that way.

Reply:
actually, fiam display already has a public umbrella file which can be used as the bridge header for this swift sample project. Removing this BridgeHeader.h file. 

thanks for the reminder.